website/docs/contributing/ADRs/ADRs.md

```diff
@@ -13,27 +13,27 @@ One such example is the decision to re-write Unleash to TypeScript.
 
 These ADRs describe decisions that concern the entire codebase. They apply to back-end code, front-end code, and code that doesn't neatly fit into either of those categories.
 
-* [Domain language](./general/ADR/domain-language.md)
+* [Domain language](./Overarching/domain-language.mddomain-language.md)
 
 ## Back-end ADRs
 
 We are in the process of defining ADRs for the back end. At the time of writing we have created the following ADRS:
 
-* [Naming](./back-end/ADR/naming.md)
-* [Preferred export](./back-end/ADR/preferred-export.md)
+* [Naming](./Back-end/naming.md)
```

Comment:
Out of curiosity, why do the directory names contain uppercase letters? Does it have to do with how they're displayed in the sidebar? In that case, I think I'd prefer to fix the casing with CSS instead of in the folder tree. Either that, or move the autogeneration one step lower, so that you explicitly declare the "Back end" and "Front end" categories, but autogenerate the content 🤔 

We generally use all-lowercase names everywhere else (I think), so I'd like to keep doing that as much as possible 💁🏼

Reply:
That's the only way I could find to fit it with the auto generation for now. Do you know the quick fix for this? I'm at the limit of my timebox for this.

Reply:
I've never used the auto-generation method, so I don't have a quick fix for it, no. What's the issue you're trying to solve? If it's sidebar casing, then we can just ignore that for now, I'd say? 🤷🏼